src/MetaMorpho.sol

```diff
@@ -345,51 +345,52 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     /// `supplied`).
```

Comment:
We should update this comment: the allocators can now choose freely a list of supply/withdraw operations to do. I think it should just mention that this is the final allocation

Reply:
Updated, is it ok?